src/rsz/README.md

```diff
@@ -297,7 +297,7 @@ repair_timing
 | `-skip_pin_swap` | Flag to skip pin swap. The default value is `False`, and the allowed values are bools. |
 | `-skip_gate_cloning` | Flag to skip gate cloning. The default value is `False`, and the allowed values are bools. |
 | `-skip_buffering` | Flag to skip rebuffering and load splitting. The default value is `False`, and the allowed values are bools. |
-| `-enable_buffer_removal` | Flag to enable buffer removal during setup fixing. The default value is `False`, and the allowed values are bools. |
+| `-skip_buffer_removal` | Flag to skip buffer removal during setup fixing. The default value is `False`, and the allowed values are bools. |
```

Comment:
This doesn't take an argument;  it is a flag.  The same for all the other -skip*

Reply:
Good catch.  Fixed